src/api.js

```diff
@@ -4,10 +4,17 @@ const ncNewsApi = axios.create({
     baseURL: "https://be-nc-news-3me1.onrender.com/api/"
 });
 
-export const getArticles = () => {
-    return ncNewsApi.get("articles").then((response) => {
-        return response.data;
-    })
+export const getArticles = (topic) => {
+    if (topic) {
+        return ncNewsApi.get(`articles?topic=${topic}`).then((response) => {
+            return response.data;
+        })
+        
+    } else {
+        return ncNewsApi.get("articles").then((response) => {
+            return response.data;
+        })
+    }
```

Comment:
This way is totally fine. Axios has its own way to deal with queries by taking an object as the second argument
`ncNewsApi.get("articles", { params: {topic, sort_by, order ...} })`
Refactoring it in the axios way to save from typing